src/types.ts

```diff
@@ -90,7 +94,7 @@ interface PluginContext<
   CtxWithPlugins extends Ctx = Ctx
 > {
   G: G;
-  ctx: CtxWithPlugins;
+  ctx: Ctx | CtxWithPlugins;
```

Comment:
Sorry, following on from @nicolodavis’s comment about `Ctx`, at the moment, `ctx` inside plugins doesn’t currently include the other plugins’ API. (More details here: https://github.com/nicolodavis/boardgame.io/issues/594#issuecomment-605820957) So these should typed as only `Ctx` too.